sarc/cli/parse/users.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+from simple_parsing import field
+
+from sarc.core.scraping.users import parse_users
+from sarc.users.db import get_user_collection
+
+
+@dataclass
+class ParseUsers:
+    from_: datetime = field(help="Start parsing the cache from the specified date")
```

Comment:
Sorry, a later comment here: it seems `datetime` is not automatically parsed. `--from_ 2020-10-01` won't work.

I guess we would need a unit test dedicated to this command `parse users`, to check command arguments.

Also, I would suggest to rename this field, for example to `--since`. It could be nicer than `--from_` which has the ending underscore :) . What do you think?